index.js

```diff
@@ -0,0 +1,53 @@
+const validatePassword = (pass) => {
+
+  const isLong = checkLength(pass)
+  const isLower = checkLower(pass)
+  const isUpper = checkCaps(pass)
+  const isNum = checkNum(pass)
+  const isSpecial = checkSpecial(pass)
+
+  if (isLong && isLower && isUpper && isNum && isSpecial) return true
+
+  return false
+}
```

Comment:
You don't need an `if` when returning true/false, the condition is already the value. You also don't need these temp variables.

```JavaScript
const validatePassword = (pass) => {
  return checkLength(pass) &&
    checkLower(pass) &&
    checkLower(pass) &&
    checkCaps(pass) &&
    checkNum(pass) &&
    checkSpecial(pass)
}
```